blueman/main/Applet.py

```diff
@@ -6,29 +6,26 @@
 from blueman.main.PluginManager import PersistentPluginManager
 from blueman.main.DbusService import DbusService
 from blueman.plugins.AppletPlugin import AppletPlugin
+from gi.repository import Gio
 import logging
 
-import gi
-gi.require_version("Gtk", "3.0")
-from gi.repository import Gtk, Gio
 
-
-class BluemanApplet:
+class BluemanApplet(Gio.Application):
     def __init__(self):
+        super().__init__(application_id="org.blueman.Applet", flags=Gio.ApplicationFlags.FLAGS_NONE)
         setup_icon_path()
 
-        check_single_instance("blueman-applet")
-
         self.plugin_run_state_changed = False
         self.manager_state = False
+        self._active = False
 
         self.Manager = Manager()
         self.Manager.connect_signal('adapter-added', self.on_adapter_added)
         self.Manager.connect_signal('adapter-removed', self.on_adapter_removed)
         self.Manager.connect_signal('device-created', self.on_device_created)
         self.Manager.connect_signal('device-removed', self.on_device_removed)
 
-        self.DbusSvc = DbusService("org.blueman.Applet", "org.blueman.Applet", "/org/blueman/applet",
+        self.DbusSvc = DbusService("org.blueman.Applet", "org.blueman.Applet", "/org/blueman/Applet",
```

Comment:
meh

Reply:
The main reason is that otherwise we end up with
``/org/blueman/applet`` for all blueman specific interface
``/org/blueman/Applet`` for all Gio.Application interfaces.

Which can be confusion, I don't care much so I can back it out.